query.js

```diff
@@ -271,7 +271,13 @@ var searchMap = {
         ]
     }),
     type: d => ({ type: d }),
-    index: d => ( { index: { $gte: parseFloat(d.startIndex), $lte: parseFloat(d.endIndex) }  }),
+    index: d => ({
+        $or: 
+        [ 
+            { index: { $in: d.startIndex.split(",").map(parseFloat) } },
```

Comment:
Let's add this so that people can also use spaces (like "1, 2, 3") and it will still work:
```suggestion
            { index: { $in: d.startIndex.split(",").map(e => e.trim()).map(parseFloat) } },
```

Reply:
That still works as far as I understand - float parsing things with trailing spaces take out the spaces